meta/autoid/autoid_test.go

```diff
@@ -120,3 +122,65 @@ func (*testSuite) TestT(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(id, Equals, int64(6544))
 }
+
+func (*testSuite) TestAsyn(c *C) {
```

Comment:
`Asyn` doesn't properly describe the test.
And add comment about the purpose of the test.